generic/tests/pktgen_perf.py

```diff
@@ -114,6 +114,7 @@ def init_vm_and_login(test, params, env, result_file, pktgen_runner):
             dev = vdpa_net_test.add_dev(params.get("netdst_nic2"), params.get("mac_nic2"))
             LOG_JOB.info("The vhost_vdpa device name is: '%s'", dev)
             LOG_JOB.info("Test vhost_vdpa with the simulator on the vm")
+            process.system_output("cat /sys/module/vdpa_sim/parameters/use_va")
```

Comment:
Is this added here for a specific reason?

Reply:
It's added to print the info so we can track any changes, but it doesn't need to be judged.